www/source/partials/docs/_using-hab-continuous-deployment.html.md.erb

```diff
@@ -39,7 +39,7 @@ Configuring a Supervisor's update strategy to point to a channel ensures that ne
 
 To start a Supervisor with an update strategy and pointing to a channel, specify them as options when starting a service.
 
-    $ hab start <origin>/<package> --strategy rolling --channel test
+    $ hab sup run <origin>/<package> --strategy rolling --channel test
```

Comment:
This has already been (correctly) changed to `hab svc load`. See https://www.habitat.sh/docs/using-habitat/.